smart-contracts/contracts/interfaces/IWETH.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.7.0 < 0.9.0;
+interface IWETH {
```

Comment:
Isn't it the same as the ERC20 interface here?

Reply:
not quite, there are deposit/withdraw functions and few others